tools/report-converter/codechecker_report_converter/analyzers/cppcheck/analyzer_result.py

```diff
@@ -50,4 +51,19 @@ def get_reports(self, analyzer_result_path: str) -> List[Report]:
                 plist_file, None, file_cache)
             reports.extend(plist_reports)
 
+        # Fix repotrs by adding the actual warining message
+        # as the last bug_path_event
```

Comment:
Did we decide adding this point to the end of the bug path finally?

Reply:
"Fix" is too broad here, could be worth an explanation **why** this is needed.

Reply:
Until we refactor the gui to indicatae the error in the location of the diagnostic message, we should add that as the last step.

Reply:
I know that, but the comment does not mention this. Also, `repotrs`!

Reply:
Yes, sorry, I've already updated the comment.